src/views/Home/components/ArticleCard/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { radius, spaces } from '@src/styles/theme'
+
+import Image from '@components/Image'
+import Text from '@components/Text'
+
+import type { News } from '../../types'
+
+const ArticleCardWrapper = styled.View`
+  margin-right: ${spaces.X_LARGE}px;
+  margin-top: ${spaces.SMALL}px;
+`
+
+const ArticleCardContent = styled.View`
+  width: 260px;
+  height: 155px;
```

Comment:
These values are not present in our style files ?

Reply:
Nop